### PR TITLE
Search bar logo fix

### PR DESCRIPTION
**Pull request by JohnRising**
Search bar logo now takes user home instead of Stackup website.

### CODE REVIEW

File: apps/web/src/components/header.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const Header = ({ backLinkUrl, backLinkLabel }) => {
               <Image
                 src="./logotype-blue-navy-32x142.png"
                 maxW="142px"
-                maxH="32x"
+                maxH="32px"
```
Comment (hazim-j) [inline apps/web/src/components/header.js:36]: nice catch!

File: apps/web/src/components/search.js [hunk 2]
```diff
@@ -48,8 +48,14 @@ export const Search = ({ onSearch, onClear }) => {
       <Box borderBottomWidth="1px">
         <HStack spacing={['8px', '16px']} py="16px" px="16px" maxW="xl" margin="0 auto">
           <LinkBox>
-            <LinkOverlay href="https://stackup.sh/" target="_blank">
-              <Image src="./mark-blue.png" maxW="32px" maxH="32x" alt="stackup logo" />
+            <LinkOverlay href="/home" target="_self">
```
Comment (hazim-j) [inline apps/web/src/components/search.js:51]: You might want to use the `next/link` for navigating within the app in this situation. Have a look here for reference: https://chakra-ui.com/docs/navigation/link-overlay#usage-with-routing-library

You can also search up `NextLink` in the code base for examples.

**Review by hazim-j — CHANGES_REQUESTED** [review]

File: apps/web/src/components/search.js (hunk 2 above)
Comment (JohnRising) [reply apps/web/src/components/search.js]: Done per https://github.com/stackupfinance/stackup/pull/94/commits/2c686c6a499b8bc3c751688aabf69647d71bccfc

**Review by hazim-j — APPROVED** [review]
LGTM

**Pull request merged**